brainfm/__init__.py

```diff
@@ -221,7 +221,21 @@ def operation(self, **kwargs):
     },
     # getMainStations  # not implemented for now, use get_stations
     # getMainStations_NoPlans  # not implemented for now, use get_stations
-    # getStationsByID  # unknown; not seen
+    "get_stations_by_id": {
+        "name": "getStationsByID",
+        "method": "post",
+        "parameters": [
+            {
+                "name": "pid",
+                "alias": "parent_id",
+                "required": True,
+                "type": int
+            }
+        ],
+        "response": jmespath.compile(
+            "{stations: stations}"),
```

Comment:
I'm also still learning jmespath but I'll give it a shot.  I remember that the indexing/list syntax was not intuitive.